vincentvanbot/model.py

```diff
@@ -0,0 +1,65 @@
+import os
+import random
+import numpy as np
+import pickle
+import matplotlib.pyplot
+from matplotlib.pyplot import imshow
```

Comment:
Let's change this to `import matplotlib.pyplot as plt` to respect the convention :)